functional_tests/keras/keras_subclassed_test_tf_2_6.yea

```diff
@@ -0,0 +1,18 @@
+id: 0.keras.subclassedmodel.tf26
+tag:
+    shard: tf26
+plugin:
+    - wandb
+command:
+    program: test_keras_subclassed_model.py
+depend:
+    requirements:
+        - pillow
+        - tensorflow>=2.6.2,<2.7
+assert:
+    - :wandb:runs_len: 1
+    - :wandb:artifacts[model-lovely-dawn-32][type]: model
+    - :op:contains:
+        - :wandb:runs[0][telemetry][3]  # feature
+        - 8  # keras
+    - :wandb:runs[0][exitcode]: 0
```

Comment:
```suggestion
    - :wandb:runs[0][exitcode]: 0

```

Reply:
You probably know this, but this flag is very useful when coming up with yea tests, so just throwing it here:
`yea --debug run functional_tests/keras/keras_subclassed_test_tf_2_6.yea`

Reply:
Thanks for reminding me about this flag.